python/GafferSceneUI/StandardAttributesUI.py

```diff
@@ -135,6 +137,24 @@ def __motionBlurSummary( plug ) :
 
 		],
 
+		# instancing
+
+		"attributes.automaticInstancing" : [
+
+			"description",
+			"""
+			By default, if Gaffer sees two objects are identical, it will pass them
+			to the renderer only once, saving a lot of memory.  You can set this to
+			false to disable that, losing this memory savings.  This can be useful
+			in certain cases like using world space displacement and wanting multiple
+			copies to displace differently.  Currently supported only by the Arnold
+			render backend.
+			""",
+
+			"layout:section", "Attributes",
```

Comment:
This feels a bit esoteric alongside fundamental things like `displayColor` and `doubleSided`, and I'm not sure the average user cares, or wants to spend the time reasoning about it (but might turn it off and pay for an expensive render unwittingly). Do you think there's any merit in putting it in in "Instancing" section all of its own?